src/package-manager.coffee

```diff
@@ -30,6 +30,7 @@ module.exports =
 class PackageManager
   constructor: ({configDirPath, @devMode, safeMode, @resourcePath}) ->
     @emitter = new Emitter
+    @hooks = new Emitter
```

Comment:
6 of 1, half dozen of the other. Your call. Either way, if people want to abuse it they will abuse it however it is named (if avoiding abuse is your intent for naming it more pedantically).